examples/single-executor/README.md

```diff
@@ -0,0 +1,15 @@
+# Single executor example
+
+This example uses the [root module](https://registry.terraform.io/modules/sourcegraph/executors/google/latest) that provisions a network, a Docker registry mirror, and a set of resources to run _one_ type of executor. To provision more than one type of executor (multiple queues or multiple environments), see the following `multiple-executors` example.
+
+The following variables must be supplied:
+
+**Note**: these variables must be set in tandem with changes to the target [Sourcegraph deployment](https://docs.sourcegraph.com/admin/deploy_executors).
+
+- `executor_sourcegraph_external_url`: The URL from which the target Sourcegraph instance is accessible from the executor instances.
+- `executor_sourcegraph_executor_proxy_password`: The shared executor password defined in Sourcegraph. The value must be the same as the `executors.accessToken` site setting described in [Configuring executors and instance communication](https://docs.sourcegraph.com/admin/deploy_executors#configuring-executors-and-instance-communication).
```

Comment:
https://docs.sourcegraph.com/admin/deploy_executors#configuring-executors-and-instance-communication points to a missing (?) heading

Reply:
Ah should just be `#installation` now